notebooks/book2/11/mcmc_gmm_demo.ipynb

```diff
@@ -1,9 +1,17 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    %%time
</pre><p>Curious -- are we making use of the time information?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/pyprobml/pull/891/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>I have removed this.</p>